src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +10,19 @@
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
+        if (email == null || password == null) {
+            return false;
+        }
+
+        if (UserService.findByEmail(email) == null) {
+            return false;
+        }
+
+        if (UserService.findByEmail(email).getPassword().equals(password)) {
+            return true;
+        }
```

Comment:
большое количество ифов - повод задуматься, не можем ли мы их как-то объединить, в данном случае, мы можем написать:
```suggestion
        User user = UserService.findByEmail(email);
        return user != null && user.getPassword().equals(password);
```
проверять почту и пароль на `null` не обязательно